chrome_extension/background.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file background.js
+ * @description Creates a new background script that listens for messages from the content script
+ * and sends them to a WebSocket server.
+*/
+
+let socket;
+
+/*
+ * Function to connect to the WebSocket server.
+*/
+function connectWebSocket() {
+  socket = new WebSocket("ws://localhost:8765");
+
+  socket.onopen = function() {
+    console.log("WebSocket connection established");
+  };
+
+  socket.onmessage = function(event) {
+    console.log("Message from server:", event.data);
+  };
+
+  socket.onclose = function(event) {
+    console.log("WebSocket connection closed", event);
+    // Reconnect after 5 seconds if the connection is lost
+    setTimeout(connectWebSocket, 5000);
```

Comment:
What do you think about defining the timeout as a named constant at the top of the file?

Reply:
Done here: c06bb4891a15e4919a4311e97a0a3ffae9bb46bd